<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户登录</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>421</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>登录</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="AdminID">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>管理员ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDLine">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordLine">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>管理员</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>普通用户</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="cno">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>借书证号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Login_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>251</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame {
	border-image:url(:/images/login.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
